subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -411,17 +418,81 @@ uint8_t ull_scan_enable(struct ll_scan_set *scan)
 			scan->ull.ticks_slot = 0U;
 		} else {
 			scan->ull.ticks_slot = ticks_interval -
-				HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_XTAL_US);
+					       ticks_slot_overhead;
 		}
 
 		lll->ticks_window = 0U;
 	}
 
-	if (IS_ENABLED(CONFIG_BT_CTLR_LOW_LAT)) {
-		ticks_slot_overhead = MAX(scan->ull.ticks_active_to_start,
-					  scan->ull.ticks_prepare_to_start);
+	handle = ull_scan_handle_get(scan);
+
+	if (false) {
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_CTLR_PHY_CODED)
+	} else if (handle == SCAN_HANDLE_1M) {
+		const struct ll_scan_set *scan_coded;
+
+		scan_coded = ull_scan_set_get(SCAN_HANDLE_PHY_CODED);
+		if (IS_PHY_ENABLED(scan_coded, PHY_CODED) &&
+		    (lll->ticks_window != 0U)) {
+			uint32_t ticks_interval_coded;
+
+			ticks_interval_coded = HAL_TICKER_US_TO_TICKS(
+				(uint64_t)scan_coded->lll.interval *
+				SCAN_INT_UNIT_US);
+			if ((ticks_interval + lll->ticks_window) ==
+			    (ticks_interval_coded +
+			     scan_coded->lll.ticks_window)) {
+				if (IS_ENABLED(CONFIG_BT_CTLR_SCAN_UNRESERVED)) {
+					scan->ull.ticks_slot = 0U;
+				} else {
+					scan->ull.ticks_slot =
```

Comment:
Commit message does not say anything about change in handling of PHY1M scanning. Why this is changed?

Reply:
Commit message updated and add comments in code.